tests/plenary/job_spec.lua

```diff
@@ -106,7 +116,7 @@ describe('Job', function()
 
       job:sync()
 
-      assert.are.same(job:result(), { 'A=100' })
+      assert.are.same(true, contains(job:result(), { 'A=100' }))
```

Comment:
Windows doesn't get a clean env. Or maybe it is a clean env but windows needs like 100 env vars to even function :rofl: Idk we just check that it contains now. Doesn't matter